model/src/channel/message/allowed_mentions/mod.rs

```diff
@@ -0,0 +1,105 @@
+use crate::id::{RoleId, UserId};
+use serde::{Deserialize, Serialize};
+
+mod builder;
+
+pub use self::builder::AllowedMentionsBuilder;
+
+#[allow(clippy::trivially_copy_pass_by_ref)]
+fn is_false(value: &bool) -> bool {
```

Comment:
We have an `is_false` in the RoleTags file, let's move that to a common place (src/util.rs? I'm not sure).